submissions/Harnytskyi/js-post-oop/index.js

```diff
@@ -0,0 +1,104 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://harnytskyi.github.io/a-tiny-JS-world
+   Web app: https://github.com/Harnytskyi/a-tiny-JS-world
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+class Inhabitant {
+    constructor(species, name, gender, saying, friends = []) {
+        this.species = species;
+        this.name = name;
+        this.gender = gender;
+        this.saying = saying;
+        this.friends = friends;
```

Comment:
Use `addFriends` method here. This way if any changes in handling of inbound friends list occur you'd need to change handling rules in a single place - in `addFriends`.

Reply:
If I understood you correctly, it should look like `this.friends = this.addFriends();` ?

Reply:
Well, not exactly this way. As far as i understand the signature of the method it requires an argument and returns nothing. Just use it the way the method should be used, just within the constructor.